### ISSUE TITLE
Game UI updates are messed up

### ISSUE THREAD

**Issue by maximtitovich** [ublue-os/bazzite#1065]
### Describe the bug

In game UI when you go to system updates, you get all steamos updates and beta updates to steam client. When installing and restarting the system, they appear again.

### What did you expect to happen?

After installing updates, they do not appear. Steam Beta Client updates do not show and install when stable channel is chosen.

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck-gnome:stable
                   Digest: sha256:abc2bd7e26359b519a2092ee8ad02893ec9fbf6ee6d4c862511846903f4a7360
                  Version: 39.20240415.0 (2024-04-16T02:23:14Z)
          LayeredPackages: openrazer-meta razergenie

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck-gnome:stable
                   Digest: sha256:abc2bd7e26359b519a2092ee8ad02893ec9fbf6ee6d4c862511846903f4a7360
                  Version: 39.20240415.0 (2024-04-16T02:23:14Z)
```

### Hardware

Jupiter

### Extra information or context

It is a little bit annoying and misleading.

**Comment by maximtitovich** [comment]
I rebased to kde version of the repo which is bazzite-deck and updates worked fine with version 40. Seems like to be an issue with gnome pre 40 image.

**Comment by Pappnose** [comment]
I have the same problem on the Legion Go. No matter if Fedora 39 KDE or now Fedora 40 KDE, extra completely reinstalled.

**Comment by ShadowFlare** [comment]
I had a similar issue recently (on the KDE bazzite-deck version, after the update to Fedora 40) where it got stuck on some version and wouldn't update, and would just tell me there was an update available every time after I tried updating and rebooting when it prompted. Even tried updating from desktop mode and I don't think it gave any errors. I did a rebase, just to stable and still bazzite-deck that I was already on, and it got the latest version. Had some issue and tried rolling back just to make sure there wasn't something wrong with the installation (the missing logo bug that was in one build), and afterward it updated fine even from the version I was on before, so I don't know what could have been the problem in my case.
